staging/src/k8s.io/api/storage/v1alpha1/types.go

```diff
@@ -25,6 +25,9 @@ import (
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:prerelease-lifecycle-gen:introduced=1.9
+// +k8s:prerelease-lifecycle-gen:deprecated=1.10
```

Comment:
I guess we never officially deprecated this. Should it be 1.21? Can you also open up a tracking issue so we can clean this up?